src/main/java/com/bina/lrsim/simulator/ParallelSimulator.java

```diff
@@ -62,7 +67,14 @@ private Worker(Simulator sim, String outDir, String movieName, SamplesDrawer sam
     public void run() {
       try {
         ThreadLocalResources.random().setSeed(seed);
-        sim.simulate(outDir, movieName, 0, samples, targetNumBases, spec, new MersenneTwister(seed));
+          /*for now, keep two parallele implementations of simulate() to make sure
```

Comment:
should not change this, since this is public. Instead, use a new Spec to specify a new output format with adapter in it, then dispatch wrt to Spec within the implementation in Simulator